### PR TITLE
Meeting manager

### PR DESCRIPTION
**Pull request by rayaviri**
add new functionality to food meeting

### CODE REVIEW

File: src/main/java/org/jala/efeeder/api/command/impl/DefaultIn.java [hunk 1]
```diff
@@ -5,18 +5,24 @@
...
 import org.jala.efeeder.servlets.websocket.avro.MessageContext;
 import org.jala.efeeder.user.User;
-
+import java.nio.charset.StandardCharsets;
 import java.sql.Connection;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.jala.efeeder.api.command.In;
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/api/command/impl/DefaultIn.java]: There are duplicated imports

Comment (Ramir0) [reply src/main/java/org/jala/efeeder/api/command/impl/DefaultIn.java]: please, sort them

File: src/main/java/org/jala/efeeder/api/command/impl/DefaultIn.java [hunk 2]
```diff
@@ -35,10 +41,12 @@ public void addParameter(String key, List<String> values) {
 
     @Override
     public String getParameter(String key) {
-        if (!parameters.containsKey(key) || parameters.isEmpty()) {
+        if(!parameters.containsKey(key) || parameters.isEmpty()) {
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/api/command/impl/DefaultIn.java]: this validation is unnecessary 
`parameters.isEmpty()`

File: src/main/java/org/jala/efeeder/foodmeeting/EditFoodMeetingCommand.java [hunk 3]
```diff
@@ -24,24 +24,25 @@
...
+    private static final String UPDATE_FOOD_MEETING_SQL = "UPDATE food_meeting SET name= ?, image_link= ?, status=?, event_date=? WHERE id= ?;";
 
     @Override
     public Out execute(In parameters) throws Exception {
         Out out = new DefaultOut();
 
         PreparedStatement stm = parameters.getConnection()
                 .prepareStatement(UPDATE_FOOD_MEETING_SQL);
-
+
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/foodmeeting/EditFoodMeetingCommand.java]: remove this tabulation character

**Review by Ramir0 — CHANGES_REQUESTED** [review]

**Review by MirkoTV — APPROVED** [review]
Looks good to me

**Review by Ramir0 — APPROVED** [review]

**Pull request merged**